scripts/python/slim_vcf2fasta_chrom.py

```diff
@@ -0,0 +1,128 @@
+'''
+slim_vcf2fasta_chrom.py -
+use a chlamy region as the 'ref' then a mutation
+matrix to determine the alternate base at each variant site
+'''
+
+import argparse
+import itertools
+
+from tqdm import tqdm
+from cyvcf2 import VCF
+import numpy as np
+
+import ANT
+
+
+def args():
+    parser = argparse.ArgumentParser(
+        description='slim vcf -> fasta w/ reference',
+        usage='python3.5 slim_vcf2fasta_chrom.py [options]')
+
+    parser.add_argument('-v', '--vcf', required=True,
+                        type=str, help='bgzipped + tabixed vcf')
+    parser.add_argument('-t', '--table', required=True,
```

Comment:
Low-priority enhancement - remove annotation table here, and have file read in a text file containing just the sequence in fasta format (single sequence, header = region, len(sequence) = 1e6)

Could have separate script that actually queries annotation table and creates sequence, while this script just reads in that text file